App.js

```diff
@@ -1,17 +1,15 @@
 import React from 'react';
-import { StyleSheet, SafeAreaView,StatusBar } from 'react-native';
+import { StyleSheet, SafeAreaView, StatusBar } from 'react-native';
 import Amplify from 'aws-amplify'
 import awsconfig from './src/aws-exports'
 import Home from './src/Home';
-import Header from './src/Header';
```

Comment:
Did you delete this file as well? Not seeing it in the diff

Reply:
Deleted it now.